### ISSUE TITLE
All passwords are visible in the database

### ISSUE THREAD

**Issue by Redsandro** [keepassxreboot/keepassxc#3452]
## Expected Behavior
Passwords hidden in main database view

## Current Behavior
All passwords are visible in main database view

## Context
Everything was as expected. Then, suddenly, at some point, all passwords in the database view were just visible. Permanently. Also after a reboot. I can't seem to undo this.

I triple-checked, and while there are options for hidden of visible passwords in the entry pane and while editing, **there isn't actually an option to display passwords in the main view. Correct?**

I'm not sure if there is **some undocumented hotkey that I pressed, or that some kind of virus** infected the passwords manager in the hopes of taking screenshots (or whatever).

Either way, ~~I'm paranoid, so~~ for security reasons I turned off the system and writing this issue from a different computer. For that reason, I cannot paste the full debug information for the affected machine at this point. If necessary, I will update in the future.

I'm hoping I did something stupid rather than this being a bug or something more serious.

## Debug Info
KeePassXC - 2.4.3
(todo: Paste debug info from Help → About here)

Operating system: Linux Mint 19.1
CPU architecture: x86_64
Kernel: 4.15.0.58

**Comment by droidmonkey** [comment]
![image](https://user-images.githubusercontent.com/2809491/63230059-c4797600-c1d5-11e9-9cde-3049f29a22a7.png)

Or simply Ctrl+Shift+C

**Comment by Redsandro** [comment]
Wow, I did not know about this shortcut.

I feel a bit silly now, but _I did actually search_ before posting, and read this:

@droidmonkey [commented](https://github.com/keepassxreboot/keepassxc/issues/2166#issuecomment-418765907):

> We DO NOT show passwords in the entry list or details panel for security reasons even when this setting is enabled.

I misread the comment to a different issue as meaning that there would be no way to show passwords, and I concluded that my issue was worthy of paranoia.

Thank you for the screenshot. It helps. :+1:
